addons/web/static/src/js/views/kanban/kanban_renderer_mobile.js

```diff
@@ -73,40 +77,46 @@ KanbanRenderer.include({
 
     /**
      * Displays the quick create record in the active column
+     * override to open quick create record in first column if active tab is Add Column
```

Comment:
Not sure to understand: this method is an override (missing `@override` annotation)? or this method could be overridden to achieve this behavior?